src/qml/SettingsPage/SettingsPage.qml

```diff
@@ -112,7 +112,7 @@ Page {
                 }
                 text: i18n.tr("Dialpad tones")
             }
-
+/*   commented for now as not fully implemented
```

Comment:
Rather than commenting out code, it's better to just delete it, especially for things that are unlikely to return. We can always look at the previous revision to find the code which was there. Another option would be instead to add in the element we want to hide below, something like:

```
    visible: false // TODO: Finish enablement of SIP Online Account first
```

preferably also with a link to an issue about the incomplete feature.